### ISSUE TITLE
support project specific settings (at least for XML formatting)

### ISSUE THREAD

**Issue by rhoehener** [eclipse-wildwebdeveloper/wildwebdeveloper#1602]
So we can put them under version control for the team:

![image](https://github.com/eclipse-wildwebdeveloper/wildwebdeveloper/assets/13407181/c07d62d9-c0cb-4e77-9ab1-18330bf9ac07)

**Comment by mickaelistria** [comment]
Good idea. Is this something you'd be interested in contributing? We can provide you some assistance to do so if needed.
